ops/test/api.tf

```diff
@@ -14,16 +11,14 @@ module "simple_report_api" {
   tenant_id        = data.azurerm_client_config.current.tenant_id
 
   app_settings = {
-    SPRING_PROFILES_ACTIVE                         = "azure-dev,okta-dev,no-okta-mgmt"
+    SPRING_PROFILES_ACTIVE                         = "azure-test,okta-test,no-okta-mgmt"
```

Comment:
As noted, there's a conflict between the env variable and what's in the profiles.include property—I don't have a strong feeling about which should be set, but definitely only one of them should.